### ISSUE TITLE
Failure on AppVeyor Python 3.5

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/spm1d-feedstock#1]
Seems there was some sort of failure in the feedstock that we didn't see at staged-recipes. Here is an example [build failure](https://ci.appveyor.com/project/conda-forge/spm1d-feedstock/build/1.0.1/job/cna323bmjyygkdp7#L1706). There has been a bunch of shuffling of Qt/PyQt stuff in the last few days. This may have had an impact, but I have not been watching this closely. As @ocefpaf has been more closely involved, am copying him on this in case he knows more.

**Comment by jakirkham** [comment]
Looking a little more closely it seem Qt5 is being pulled in again for some reason. Would expect that adding a pinning of some sort to Qt4 again would fix this. Though I thought `matplotlib` already had such a pinning. Perhaps a non-pinned `matplotlib` package is being pulled in instead?

**Comment by melund** [comment]
Yes. I had pyqt pinned at some point in the staged recipe. But when we removed it again it seemed like the issue was sorted. Do you suggest we pin it again or wait until it is fixed for matplotlib?

**Comment by jakirkham** [comment]
Not sure. Let's see what @ocefpaf says.

**Comment by melund** [comment]
ping @ocefpaf

**Comment by melund** [comment]
Ping @jakirkham
